test/100-basic-name-resolution.bats

```diff
@@ -53,15 +53,15 @@ load helpers
 	gw=$(echo "$config_a1" | jq -r .network_info.podman1.subnets[0].gateway)
 	create_container "$config_a1"
 	a1_pid=$CONTAINER_NS_PID
-	run_in_container_netns "$a1_pid" "dig" "+short" "aone" "@$gw"
+	run_in_container_netns "$a1_pid" "dig" "+short" "aone" "@$gw" "AAAA"
 	assert "$ip_a1"
 	# Set recursion bit is already set if requested so output must not
 	# contain unexpected warning.
 	assert "$output" !~ "WARNING: recursion requested but not available"
 
-	run_in_container_netns "$a1_pid" "dig" "+short" "google.com" "@$gw"
-	# validate that we get an ipv4
-	assert "$output" =~ "[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+"
+	run_in_container_netns "$a1_pid" "dig" "+short" "google.com" "@$gw" "AAAA"
+	# validate that we get an ipv6
+	assert "$output" =~ "[0-9a-fA-F]+\:[0-9a-fA-F]+\:[0-9a-fA-F]+\:[0-9a-fA-F]+"
```

Comment:
I am confused about this regex? It doesn't actually validate a proper ipv6. You could have an address like `fd11::1`

Reply:
If we have ipcalc installed in the VMs we can use: `ipcalc -c $ip`

Reply:
I think it should be installed by default, I can amend test and see if it passes otherwise i can add a `TODO` comment here and once automation image will get updated we can replace the test with `ipcalc`